alert-policies/activemq/HeapMemoryUsage.yml

```diff
@@ -0,0 +1,27 @@
+name: Heap Memory Usage
+
+description: |+
+  This alert is triggered when the Heap Memory usage used  is higher than 100GB during 5 minutes.
```

Comment:
OLD:
This alert is triggered when the Heap Memory usage used  is higher than 100GB during 5 minutes.
Update needed to:
This alert is triggered when the used heap memory usage is higher than 100GB during 5 minutes.

Reply:
Hi @pkosanam,
Updated grammar mistake changes.